extensions/ql-vscode/src/query-results.ts

```diff
@@ -200,10 +200,13 @@ export class LocalQueryInfo {
     private cancellationSource?: CancellationTokenSource, // used to cancel in progress queries
     public failureReason?: string,
     public completedQuery?: CompletedQueryInfo,
-    public evalLogLocation?: string,
-    public evalLogSummaryLocation?: string,
-    public jsonEvalLogSummaryLocation?: string,
-    public evalLogSummarySymbolsLocation?: string,
+    public evalutorLogPaths: EvaluatorLogPaths = {
+      log: undefined,
+      humanReadableSummary: undefined,
+      endSummary: undefined,
+      jsonSummary: undefined,
+      summarySymbols: undefined,
+    },
```

Comment:
Could this be `undefined` by default instead? That would make the change for making `log` `| undefined` unnecessary as well.